pages/TheExplorationLibrary.html

```diff
@@ -36,7 +36,7 @@ <h2>
     the gallery shows only mathematics explorations.
   </p>
 
-  <img src="static/images/libraryCategoryHighlighted.png" width="100%">
+  <img src="static/images/libraryCategoryHighlighted.png" width="100%" alt="Total eight 'mathematics' exploration cards are present in two rows, with four cards in each row.">
```

Comment:
Eight 'mathematics' exploration cards shown in two rows, with four cards per row. The cards display topics such as AAA, BBB and CCC.